x-pack/plugins/security_solution/cypress/screens/create_new_rule.ts

```diff
@@ -94,6 +94,9 @@ export const AT_LEAST_ONE_VALID_MATCH = 'At least one indicator match is require
 
 export const AT_LEAST_ONE_INDEX_PATTERN = 'A minimum of one index pattern is required.';
 
+export const FORBIDDEN_INDEX_PATTERN =
+  'The index pattern can not be *. Please create a different index pattern.';
```

Comment:
```suggestion
  'The index pattern cannot be *. Please choose a more specific index pattern.';
```

As a user, my first question is going to be _why_ it can't be `*`.